From b137ca16b54c67d76714ea5a0138741959b0dc29 Mon Sep 17 00:00:00 2001
From: David Bauer <mail@david-bauer.net>
Date: Mon, 13 Jul 2020 23:37:37 +0200
Subject: [PATCH] scripts: remove dependency on swig

Don't build the libfdt tool, as it has a dependency on swig (which
OpenWrt does not ship).

This requires more hacks, as of-platdata generation does not work
without it.

Signed-off-by: David Bauer <mail@david-bauer.net>
---
 scripts/dtc/Makefile | 2 --
 1 file changed, 2 deletions(-)

--- a/scripts/dtc/Makefile
+++ b/scripts/dtc/Makefile
@@ -18,5 +18,3 @@ HOSTCFLAGS_dtc-parser.tab.o := -I$(src)
 # dependencies on generated files need to be listed explicitly
 $(obj)/dtc-lexer.lex.o: $(obj)/dtc-parser.tab.h
 
-# Added for U-Boot
-subdir-$(CONFIG_PYLIBFDT) += pylibfdt
